api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/NewClaimService.java

```diff
@@ -106,19 +108,25 @@ private Claim buildClaim(Claimant claimant, EligibilityResponse eligibilityRespo
                 .build();
     }
 
-    private ClaimResult createResult(Claim claim, EligibilityResponse eligibilityResponse) {
-        Optional<VoucherEntitlement> entitlement = eligibilityResponse.getEligibilityStatus() == EligibilityStatus.ELIGIBLE
-                ? Optional.of(getEntitlement(claim, eligibilityResponse))
-                : Optional.empty();
+    private ClaimResult createResult(Claim claim) {
+        return ClaimResult.builder()
+                .claim(claim)
+                .voucherEntitlement(Optional.empty())
+                .build();
+    }
+
+    private ClaimResult createResult(Claim claim, PaymentCycleVoucherEntitlement voucherEntitlement) {
+        VoucherEntitlement firstVoucherEntitlement
+                = min(voucherEntitlement.getVoucherEntitlements(), Comparator.comparing(VoucherEntitlement::getEntitlementDate));
```

Comment:
As per my other comment below, if you actually stored the `VoucherEntitlement`s by date then you wouldn't need to do this, this is quite confusing!

Reply:
As discussed, we'll do this in the next PR.